portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,11 +68,26 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String comment = getParameter(request, "text-input", "");
+    String name = getParameter(request, "name-input", "");
+    String message = getParameter(request, "text-input", "");
+    long timestamp = System.currentTimeMillis();
+
+    Date date = new Date(timestamp);
+    SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");
```

Comment:
You can make the dateFormat a `static final` class level variable since it is reusable.

Reply:
When I put 'static final' together there were errors, so I just used 'final' if that is okay.